src/main/scala/com/codacy/client/bitbucket/v2/service/PullRequestServices.scala

```diff
@@ -68,19 +73,22 @@ class PullRequestServices(client: BitbucketClient) {
     client.postJson(Request(url, classOf[JsObject]), JsNull)
   }
 
-  def createComment(author: String, repo: String, prId: Int, body: String,
+  def createLineComment(author: String, repo: String, prId: Int, body: String,
```

Comment:
I think commit comment would be better

Reply:
This happens in package v1 as well, should I also change it there?